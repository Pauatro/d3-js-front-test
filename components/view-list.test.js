import createViewList from './view-list'

it('renders correctly', () => {

    const container = document.createElement('div')
    container.style = "width: 300px; height: 200px;"
    container.id = "target"

    document.body.append(container)

    createViewList("target")
    expect(container).toMatchSnapshot();
});