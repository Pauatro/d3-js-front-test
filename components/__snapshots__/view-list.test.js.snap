// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="target"
  style="width: 300px; height: 200px;"
>
  <div
    class="view-list"
  />
</div>
`;
